### ISSUE TITLE
about dssm loss

### ISSUE THREAD

**Issue by JepsonWong** [PaddlePaddle/PaddleRec#317]
https://github.com/PaddlePaddle/PaddleRec/blob/5cd4ae7e8da86262bc379b253d9f6cf83c4a7786/models/match/dssm/train.py#L53

感觉这里应该改为下面的代码：如果不加`axis`参数，后续的`paddle.mean`函数将没有任何意义，而且得到的`avg_cost`将会是一个batch中每条样本的`损失之和`，而不是`损失平均`。

```
loss = -paddle.sum(paddle.log(hit_prob), axis=-1)
avg_cost = paddle.mean(x=loss)
```

**Comment by JepsonWong** [comment]
@yinhaofeng

**Comment by yinhaofeng** [comment]
感谢您的建议，我们已经按照建议修改了
